butane_cli/src/main.rs

```diff
@@ -113,6 +113,20 @@ However if the migration has been manually edited, it will need to be manually r
     let args = app.get_matches();
     let mut base_dir = args.get_one::<PathBuf>("path").unwrap().clone();
     base_dir.push(".butane");
+
+    // Prevent 'makemigration' when there is a detached migration.
+    match args.subcommand() {
+        Some(("makemigration", _)) => {
+            if let Ok(ms) = get_migrations(&base_dir) {
+                if ms.has_detached_migration().unwrap_or(false) {
+                    eprintln!("Found detached migration. Please delete or manually re-attach it.");
```

Comment:
Since `butane` doesn't provide any help in that, we should probably print out the file path to the detached migration so the user at least knows where it is.